lib/hop_swipe_cards.dart

```diff
@@ -130,23 +130,23 @@ class _HopSwipeCardsState extends State<HopSwipeCards>
   Alignment frontCardAlign = Alignment.center;
   Alignment afterSwipeAlignment = Alignment.center;
 
-  AnimationController _animationController;
+  AnimationController? _animationController;
```

Comment:
You can use `late final` keyword for initialization of AnimationController instead of nullable.

Flutter team also migrated animation related library to null-safety as following.
#### before:
- https://github.com/flutter/flutter/blob/9b2d32b605/packages/flutter/lib/src/widgets/implicit_animations.dart#L366
 
#### after:
- https://github.com/flutter/flutter/blob/371dd87aa078467913db65a9d83b2915b34886ac/packages/flutter/lib/src/widgets/implicit_animations.dart#L357

Reply:
Thank you very much for the information,i was wondering about the same 🙏 
I fixed it